### DISCUSSION TITLE
[Feature] Share People Automatically

### DISCUSSION THREAD

**Discussion by jdguillot** [immich-app/immich#3384] [Feature Request]
### The feature

I would like the ability to share individual people to other users within my Immich. This way when I take a photo of my daughter for instance, it would be automatically shared to my wife.

This is an example of how Google Photos does it.
![image](https://github.com/immich-app/immich/assets/7525038/e0bff46b-11ec-47a8-8771-79c08bc018f2)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #3306
